src/roles.json

```diff
@@ -1738,6 +1738,19 @@
     "reminders": [],
     "setup": false,
     "ability": "Once per day, you may choose to kill an alive neighbour, if your other alive neighbour agrees."
+  },
+  {
+    "id": "organgrinder",
+    "name": "Organ Grinder",
+    "edition": "",
+    "team": "minion",
+    "firstNight": 0,
+    "firstNightReminder": "",
+    "otherNight": 0,
+    "otherNightReminder": "",
+    "reminders": ["About to die"],
+    "setup": false,
+    "ability": "All players keep their eyes closed when voting & the vote tally is secret. Votes for you only count if you vote."
```

Comment:
Just check the `&` symbol gets translated as text directly, HTML treats it as a special char so sometimes if its pasting HTML directly you need `&amp;` or something instead. Worth checking the other string messages too

Reply:
![image](https://user-images.githubusercontent.com/67415045/231397118-dfd08f1a-aaa9-4ead-9d1f-ad7fa0137a03.png)
![image](https://user-images.githubusercontent.com/67415045/231397290-ccd61fd8-ed64-4137-905a-68dddab5e1f2.png)

All good it seems

Reply:
👍